addons/onedrive/client.py

```diff
@@ -54,10 +59,54 @@ def user_info_for_token(self, access_token):
         :rtype: dict
         :return: a dict containing metadata about the token's owner.
         """
-        return self._make_request(
+
+        # get user properties from /me endpoint
+        me_url = self._build_url(settings.MSGRAPH_API_URL, 'me')
+        me_resp = self._make_request(
             'GET',
-            self._build_url(settings.MSLIVE_API_URL, 'me'),
-            params={'access_token': access_token},
+            me_url,
+            headers=self._default_headers,
             expects=(200, ),
             throws=HTTPError(401)
-        ).json()
+        )
+        me_data = me_resp.json()
+        logger.debug('me_data:({})'.format(me_data))
```

Comment:
ditto, extra log or left-over? if intentional, is logging full response body necessary?

Reply:
Same as above.